### PR TITLE
Add Logger module

### PR DESCRIPTION
**Pull request by martindsq**
I created a file in the architectural decisions to explain why I decided to use swift packages instead of frameworks.
This PR adds the Logger package, removes cocoalumberjack from the pod file, and almost no changes in the code were made as I added some functions in the Logger package to keep the signature of the functions exactly the same as before (just needed to import the Logger module almost everywhere).

### CODE REVIEW

**Review by mplorentz — CHANGES_REQUESTED** [review]
Hey Martin, awesome work here. I think this is a great pattern for how we can abstract some of our major app components into their own silos. I really love the way the tests came together. I fixed them by adding the Logger in the "Link Binary With Libraries" build phase in the Unit Tests target. 

I can run the Logger tests by selecting the Logger scheme in Xcode. They don't run when I run the old Unit Tests target. I think I like this, because unless I change the Logger package I don't need to spend time running those tests. However it doesn't look like Github Actions are running the new Logger tests. Could you update `.github/workflows/main.yml` so that they do? Also I turned on code coverage reports, random test order, and parallel execution for the Logger tests. You managed to get 70% coverage of the logger package!

Also I see your Domain/Delivery/Infrastructure pattern, but I'm not sure I understand what sorts of things go in the Infrastructure folder. Is there a blog post or something that explains this pattern? Maybe that would be another good thing to include in the ADR.

File: Logger/Sources/Domain/Services/LoggerServiceAdapter.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+//  LoggerServiceAdapter.swift
+//  
+//
+//  Created by Martin Dutra on 10/2/22.
+//
+
+import Foundation
+import os.log
+
+class LoggerServiceAdapter: LoggerService {
```
[the remark this answers is not in the document]
Comment (martindsq) [reply Logger/Sources/Domain/Services/LoggerServiceAdapter.swift:15]: Added a comment on this class, and mainly on Log class in the Delivery folder

File: Resources/Settings.bundle/Acknowledgements.plist [hunk 2]
```diff
@@ -50,30 +50,6 @@ THE SOFTWARE.</string>
 			<key>Type</key>
 			<string>PSGroupSpecifier</string>
 		</dict>
-		<dict>
```
[the remark this answers is not in the document]
Comment (martindsq) [reply Resources/Settings.bundle/Acknowledgements.plist:53]: Added CocoaLumberjack back to the Acknolwedgements

File: Logger/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Logger
+
+A description of this package.
```
[the remark this answers is not in the document]
Comment (martindsq) [reply Logger/README.md]: Added a README

File: Architecture/0005-add-logger-module [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+The idea is to modularize the source code of Planetary in different modules to keep the code more organized and prevent dependencies from permeate different parts of the code.
+
+## Decision
+
+The following guide was followed: https://developer.apple.com/documentation/swift_packages/organizing_your_code_with_local_packages. Swift packages will be used to create modules.
+
+## Consequences
+
+- No impact in performance is expected.
```
[the remark this answers is not in the document]
Comment (martindsq) [reply Architecture/0005-add-logger-module]: Wrote some consequences

**Review by mplorentz — APPROVED** [review]
This looks great! I am reading the articles you sent on IDD and liking them. I want to discuss your vision there more in the coming days, but I don't think we need to wait on that to merge this.

**Pull request merged**